fastapi_pagination/ext/motor.py

```diff
@@ -26,7 +26,7 @@ async def paginate(
     total = await collection.count_documents(query_filter)
     cursor = collection.find(query_filter, skip=raw_params.offset, limit=raw_params.limit, **kwargs)
     if sort is not None:
-        cursor = cursor.sort(*sort)
+        cursor = cursor.sort(sort)
```

Comment:
I think it is required to check instance of sort. in case it is tuple, then unpack, otherwise pass as is.